User-migration/ImportUsers.ps1

```diff
@@ -0,0 +1,73 @@
+﻿Add-PSSnapin Microsoft.SharePoint.PowerShell -erroraction SilentlyContinue
+$logFileName="Log $(get-date -f _yyyy-MM-dd_HH_mmss).txt"
+Start-Transcript -path $logFileName -append
+
+######################################Config Section#########################################################
+# Import CSV file name
+$UserListCSV = "Members.csv"
+# Digital.ai IDP URL
+$Uri = 'https://XXXXXXXX/identity/v1/users'
+# Account ID - User need to be added
+$account_ID = "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+# Token ID
+$Token= "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+#############################################################################################################
+ 
+# Import the CSV file
+$UserList = Import-CSV $UserListCSV #-header("GroupName","UserAccount") - If CSV doesn't has headers
+
+function Create-User {
+    param(
+        $email,
+        $family_name,
+        $given_name,
+        $username
+    )
+    
+    $body = @"
+{ 
+    "account_id": "$account_ID",
+    "email": "$email",
+    "given_name": "$given_name",
+    "family_name": "$family_name",
+    "groups": [],
+    "is_password_temporary": true,
+    "roles": [],
+    "send_password_reset": true,
+    "username": "$username"
+} 
+"@ 
+
+    $headers = @{
+        "Content"       = "application/json"
+        "Authorization" = "Bearer " +$token
+    }
+
+    try {
+        Invoke-RestMethod -Uri $uri -Method Post -Headers $headers -Body $body -ContentType "application/json"
```

Comment:
Unfamiliar with `Invoke-RestMethod` ... what HTTP codes will be considered happy versus throwing an exception?  (Identity returns a 201 on success - some tools only recognize 200 as "success".)